### PR TITLE
Ai starting on UI

### PR DESCRIPTION
**Pull request by aizatibraimova**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -8,9 +8,25 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
+
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:12]: Great job putting your CDN assets in a partial, this helps with adding new assets and not having everything inside your layout!

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (JoseMDO) [inline app/controllers/application_controller.rb:2]: Nice!

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -8,9 +8,25 @@
...
+
   </head>
 
   <body>
-    <%= yield %>
+
+    <%= render partial: "shared/navbar" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:18]: Same here with the navbar 👍

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -8,9 +8,25 @@
...
+
+    <div class="container">
+      <% if notice.present? %>
+        <%= render partial: "shared/flash", locals: { message: notice, css_class: "success" } %>
+      <% end %>
+
+      <% if alert.present? %>
+        <%= render partial: "shared/flash", locals: { message: alert, css_class: "danger" } %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:26]: Same with the flash messages :)

File: lib/tasks/dev.rake [hunk 5]
```diff
@@ -10,6 +11,21 @@ task sample_data: :environment do
...
+  usernames << "bob"
+
+  usernames.each do |username|
+    User.create(
+      email: "#{username}@example.com",
+      password: "password",
+      username: username.downcase,
+      private: [true, false].sample,
+    )
+  end
```
Comment (JoseMDO) [inline lib/tasks/dev.rake:26]: Great way to create sample data!

**Review by jniewiel — APPROVED** [review]
Looks good!

**Review by JoseMDO — APPROVED** [review]

**Pull request merged**